.github/actions/lfs-copy/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
       uses: actions/cache@v3
       with:
         path: lfs_data
-        key: lfs-data-${{inputs.lfs_sha}}-1
+        key: lfs-data-${{inputs.lfs_sha}}-0
```

Comment:
ok to reset to 0 but I dont see why this change would be needed

Reply:
it's not, I'll revert